### DISCUSSION TITLE
Animation/Loaders to splash screen

### DISCUSSION THREAD

**Discussion by cohenadir** [buildnatively/community#67] [Ideas]
### Use case

```Text
first loading screen takes a long time to load
```

### Idea

```Text
To improve the user experience, you might want to add a custom gif or loader/selection from a database
```

**Comment by romanfurman6** [comment]
Hello @cohenadir, we've added the ability to include a GIF loader in the loading screen.

![SCR-20240109-kqa](https://github.com/buildnatively/community/assets/22365403/a8998101-bc90-4fa1-8f22-c4dc529ed8f3)
